src/run-es.js

```diff
@@ -63,7 +72,7 @@ class Elastic {
   async upgradeReindex(index) {
     try {
       const indexSettings = await this.es.getSettings(index);
-      if (!indexSettings.hasOwnProperty(index)) return;
+      if (!Object.prototype.hasOwnProperty.call(indexSettings, index)) return;
```

Comment:
What's the difference between these two?
Unless `hasOwnProperty` was overwritten on `indexSettings`, I don't see a difference

Reply:
this was b/c of eslint - https://eslint.org/docs/rules/no-prototype-builtins ... eslint was complaining about `hasOwnProperty()` :-\ ... do you recommend sticking to `hasOwnProperty()` and turning off the rule in eslintrc?

Reply:
If it's within the godaddy eslint rules, I would keep it for consistency, otherwise I would remove it.
If it's kept, I would make a util method wrapping this